lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -1,37 +1,40 @@
-;(function (define, undefined) {
+;(function(define, undefined) {
     'use strict';
     define([
-        'gettext', 'jquery', 'underscore', 'backbone', 'js/views/fields', 'js/views/image_field', 'backbone-super'
-    ], function (gettext, $, _, Backbone, FieldViews, ImageFieldView) {
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/views/fields', 'js/views/image_field',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'backbone-super'
+    ], function(gettext, $, _, Backbone, FieldViews, ImageFieldView, HtmlUtils) {
 
         var LearnerProfileFieldViews = {};
 
         LearnerProfileFieldViews.AccountPrivacyFieldView = FieldViews.DropdownFieldView.extend({
 
-            render: function () {
+            render: function() {
                 this._super();
                 this.showNotificationMessage();
                 this.updateFieldValue();
                 return this;
             },
 
-            showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+            showNotificationMessage: function() {
+                var accountSettingsPageUrl = this.options.accountSettingsPageUrl,
+                    accountSettingsLink = HtmlUtils.HTML(
```

Comment:
This requires nested `interpolateHtml`.  You want the text and the url to be HTML escaped.